core/er-coap-13/er-coap-13.c

```diff
@@ -696,7 +777,7 @@ coap_parse_message(void *packet, uint8_t *data, uint16_t data_len)
         ++current_option;
       }
     }
-    while (x!=(unsigned int *)&option_length && (x=(unsigned int *)&option_length));
+    while (x != &option_length && (x = &option_length));
```

Comment:
Paranthesis

Reply:
It is an assign and my linter (and possible compiler) does complain if I remove the parenthis, no changes.